st2common/st2common/transport/bootstrap_utils.py

```diff
@@ -69,3 +72,12 @@ def wrapped_register_exchanges(connection, channel):
                                       retry_wrapper=retry_wrapper)
 
         retry_wrapper.run(connection=conn, wrapped_callback=wrapped_register_exchanges)
+
+
+def register_exchanges():
+    retrying_obj = retrying.Retrying(
```

Comment:
Per in-person discussion this change request would lose consistency with existing functionality with `db_setup` so we'll keep it the same.